zombienet/bridge-hubs/bridge_hub_rococo_local_network.toml

```diff
@@ -48,19 +48,19 @@ cumulus_based = true
 	]
 
 	# run bob as parachain collator
-	[[parachains.collators]]
-	name = "bob-collator"
-	validator = true
-	command = "{{POLKADOT_PARACHAIN_BINARY_PATH}}"
-	rpc_port = 8934
-	ws_port = 8944
-	args = [
-		"-lparachain=trace,runtime::bridge-hub=trace,runtime::bridge=trace,runtime::bridge-dispatch=trace,bridge=trace,runtime::bridge-messages=trace,xcm::weight=trace,xcm::filter_asset_location=trace,xcm::send_xcm=trace,xcm::barriers=trace,xcm::barrier=trace,xcm::execute_xcm=trace,xcm::contains=trace,xcm::execute_xcm_in_credit,xcm::process_instruction=trace,xcm::currency_adapter=trace,xcm::origin_conversion=trace,xcm::fungibles_adapter=trace,xcm::process=trace,xcm::execute=trace,xcm::should_execute=trace",
-	]
-	extra_args = [
-		"--force-authoring", "--no-mdns", "--bootnodes {{'alice-collator'|zombie('multiAddress')}}",
-		"-- --port 41334 --rpc-port 48934 --ws-port 48944 --no-mdns", "--bootnodes {{'bob-validator'|zombie('multiAddress')}}"
-	]
+#	[[parachains.collators]]
```

Comment:
Maybe just some test comment, or you actually had to comment it out?

Reply:
reverted,
I had some flaky problems run it locally, but then I realized my polkadot-parachain-mint was build in debug mode (6GB binary) :), that caused problems